server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -14,41 +14,110 @@
 
 package com.google.coffeehouse.common;
 
+import static com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+
 import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.Mutation;
 import com.google.cloud.spanner.ResultSet;
 import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.cloud.spanner.Value;
+import java.util.ArrayList;
+import java.util.List;
 
 /**
 * The StorageHandler class holds the query functions to get information from the database.
 */
 public class StorageHandler {
   public static final String NO_PRONOUNS = "No pronouns";
   public static final String PERSON_DOES_NOT_EXIST = "This person does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_PERSON = "ERROR: More than one person per user ID.";
+  public static final String ERROR_MORE_THAN_ONE_PERSON = "More than one person per user ID.";
 
   public static final String CLUB_DOES_NOT_EXIST = "This club does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_CLUB = "ERROR: More than one club per club ID.";
+  public static final String ERROR_MORE_THAN_ONE_CLUB = "More than one club per club ID.";
 
   public static final String NO_AUTHOR = "No author";
   public static final String NO_ISBN = "No ISBN";
   public static final String BOOK_DOES_NOT_EXIST = "This book does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_BOOK = "ERROR: More than one book per book ID.";
+  public static final String ERROR_MORE_THAN_ONE_BOOK = "More than one book per book ID.";
+
+  /**
+  * Adds a membership from the database.
+  * This method checks if a person is already a member of a club. If the person is already a member, it will throw
+  * an exception. Otherwise, it adss the row containing the user ID and club ID to the Memberships table.
+  *
+  * @param  dbClient    the database client 
+  * @param  userId      the user ID string used to insert the membership into the table
+  * @param  clubId      the club ID string used to insert the membership into the table
+  */
+  public static void addPersonClubMembershipMutation(DatabaseClient dbClient, String userId, String clubId) {
+    List<Mutation> mutations = new ArrayList<>();
+    if (StorageHandlerHelper.checkPersonInClubQuery(dbClient, userId, clubId)) {
```

Comment:
The DB itself is structured so that the primary keys of the memberships table are the user ID and club ID (since the combination of both will be unique to every entry in the DB), and therefore a membership could not get added twice.